.github/workflows/master.yml

```diff
@@ -247,6 +247,14 @@ jobs:
   #     build_name: binary_s390x
   #     data: ${{ needs.RunConfig.outputs.data }}
   #     checkout_depth: 0
+  BuilderBinLOONGARCH64:
+    needs: [RunConfig, BuilderDebRelease]
```

Comment:
We need to have loongarch64 run in the CI infrastructure, otherwise the chance that the builds breaks is too big.

There are more changes required to enable such builds.
Please take a look at [this PR](https://github.com/ClickHouse/ClickHouse/pull/59870/commits/116393c9516d7ea33544dd06488f07bcd5ca7c9b) of mine (which disables s390x builds) to find the relevant places.